spec/factories/user.rb

```diff
@@ -0,0 +1,13 @@
+FactoryGirl.define do
+
+    factory :user do
+      email    'alice@example.com'
+      username 'Alicem'
+      password '12345678'
+      password_confirmation '12345678'
+
+      factory :failed_user do
+      password_confirmation '1234556789'
+    end
```

Comment:
Lint/BlockAlignment: `end` at 11, 4 is not aligned with `factory :failed_user do` at 9, 6